addons/product/product.py

```diff
@@ -470,6 +470,74 @@ def _get_product_variant_count(self, cr, uid, ids, name, arg, context=None):
             res[product.id] = len(product.product_variant_ids)
         return res
 
+    def _get_product_template_standard_price(self, cr, uid, ids, name, arg, context=None):
```

Comment:
Seems like there is place to have a generic method here, taking the field name as argument.

And also maybe we can batch
res = dict.fromkey(ids, 0.0)
unique_templates = [template for template in self.browse() if template.variant_count == 1]
res.update(dict((template.product_variant_ids[0].id, template.product_variant_ids[0].standard_price) for template in unique_templates))